### PR TITLE
Update delete command in UG to reflect changes in v1.2

### PR DESCRIPTION
**Pull request by Neethesh26**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -168,8 +168,9 @@ Format: `delete <name>`
 
 * Delete the entry of the person who has the given name.
 * Will not perform any operation if the name of the person does not exist.
-
-An optional argument `add` can be specified to complete erase all data in the book.
+* The `<name>` portion has to be the persons First or Last name in complete.
+* If the SectresBook contains persons with the same first or last name that has been input,
+the delete command will not execute but will return a list of all people with the given name.
 
 Format: `delete all`
```
Comment (rui-han-crh) [inline docs/UserGuide.md:175]: I think this line might need to be deleted too

Comment (Pinran-J) [reply docs/UserGuide.md:175]: ^ Dont think we plan to implement this anymore

Comment (Neethesh26) [reply docs/UserGuide.md]: Okay removed please check

**Review by rui-han-crh — APPROVED** [review]
thanks, lgtm

**Review by ryanczx — APPROVED** [review]
lgtm

**Pull request merged**